lib/ssl/src/tls_connection.erl

```diff
@@ -604,8 +604,11 @@ init({call, From}, {start, Timeout},
     %% Update pre_shared_key extension with binders (TLS 1.3)
     Hello1 = tls_handshake_1_3:maybe_add_binders(Hello, TicketData, HelloVersion),
 
+    MaxFragEnum = maps:get(max_frag_enum, Hello1#client_hello.extensions, undefined),
+    ConnectionStates1 = ssl_record:set_max_fragment_length(MaxFragEnum, ConnectionStates0),
+
```

Comment:
Can it be a problem if ClientHello is fragmented? @IngelaAndin

Reply:
No. Hello fragmentation is required by TLS, as noted in RFC 6066:

> Note that TLS already requires clients and servers to support fragmentation of handshake messages.

Reply:
@dotsimon  is correct all handshake messages can be fragmented regardless of the max_fragment_length extension.